src/visualizations/graph_visuals.py

```
@@ -1,6 +1,8 @@
-import plotly.graph_objects as go
+from typing import List
```

Ruff (0.14.3)

`typing.List` is deprecated, use `list` instead

(UP035)